Appraisals

```diff
@@ -11,10 +19,27 @@ end
 
 alias original_appraise appraise
 
+REMOVED_GEMS = {
+  :check => [
+    'rbs',
+    'steep',
+  ],
+}
```

Comment:
For appraisals, I think we can remove `sorbet` and `spoom` (and as a bonus we avoid flip-flopping back and forth when we run the scripts on x86 or ARM).

Also, perhaps we can remove a few more, to avoid adding yet more noise to appraisals, since we have the mechanism anyway:
* `pimpmychangelog`
* `yard`
* `rake-compiler`
* `rubocop`, `rubocop-packaging`, `rubocop-performance`, `rubocop-rspec`

Reply:
I wholeheartedly agree.

(Correct me if the following assumption is wrong) the use case we have for appraisals is to run specs, so there should be no other gem than those needed to run those specs.

`sorbet` and `spoom` I overlooked because of running this on `aarch64` where they are conditioned out.

Reply:
> (Correct me if the following assumption is wrong) the use case we have for appraisals is to run specs, so there should be no other gem than those needed to run those specs.

Yup, exactly!